schemas/in-network-rates/README.md

```diff
@@ -119,6 +119,31 @@ In situation where there is not expiration date ([see discussion here](https://g
 
 For `service_code`, if a negotiated rate for either "professional" or "institutional" `billing_class` is the same for all `service_code`s, the custom value of `CSTM-00` can be used to avoid listing all possible service codes.
 
+```json
+{
+ "negotiation_arrangement": "ffs",
+ "name": "Knee Replacement",
+ "billing_code_type": "CPT",
+ "billing_code_type_version": "2020",
+ "billing_code": "27447",
+ "description": "Arthroplasty, knee condyle and plateau, medial and lateral compartments",
+ "negotiated_rates": [{
+   "provider_groups": [{
+     "npi": [6666666666],
+     "tin":{
+       "type": "npi",
+       "value": "6666666666"
+     }
+   }],
+   "negotiated_prices": [{
+     "negotiated_type": "negotiated",
+     "negotiated_rate": 123.45,
+     "expiration_date": "2022-01-01",
+     "service_code": ["CSTM-00"],
+     "billing_class": "professional"
+   }]
+ }
```

Comment:
I think you're just missing a closing `}]` for `negotiated_rates` here :)

Reply:
Thanks!